exchanges/bitmex/bitmex_websocket.go

```diff
@@ -141,338 +152,321 @@ func (b *Bitmex) wsReadData() {
 }
 
 func (b *Bitmex) wsHandleData(respRaw []byte) error {
-	quickCapture := make(map[string]interface{})
-	err := json.Unmarshal(respRaw, &quickCapture)
+	// We don't need to know about errors, since we're using looking optimistically into the json
```

Comment:
It looks like there are missing words, or I've had too much caffeine and can't parse your comment
Plus you define and look at the error

Perhaps this rephrasing is for below?

Reply:
Fixed 5d0bc5d2